File: ALGORITHM/programmers/2023/02/삼총사.js
function solution(number) {
  let ans = 0;
  const N = number.length;
  for (let i = 0; i < N - 2; i++) {
      for (let j = i + 1; j < N - 1; j++) {
          for (let k = j + 1; k < N; k++) {
              if (number[i] + number[j] + number[k] === 0) ans++
          }
      }
  }
  return ans
}

File: ALGORITHM/programmers/2023/02/삼총사.py
def solution(number):
    ans = 0
    N = len(number)
    for i in range(N - 2):
        for j in range(i + 1, N - 1):
            for k in range(j + 1, N):
                if number[i] + number[j] + number[k] == 0:
                    ans += 1
    return ans